great_ape_safe/great_ape_safe.py

```diff
@@ -90,7 +94,7 @@ def take_snapshot(self, tokens):
         self.snapshot = pd.DataFrame(df)
 
 
-    def print_snapshot(self):
+    def print_snapshot(self, csv_destination):
```

Comment:
needs to be declared with a default value, eg `csv_destination=None`. otherwise all calls in existing code base (eg `safe.print_snapshot()` will fail since they do not provide a value for this (optional) argument.

Reply:
pushed in last commit